README_BUILD.md

```diff
@@ -89,11 +89,11 @@ this is invoked as "-DLOG_LEVEL=Debug".
 
 Cmake Options     | Default | Description                                    |
```

Comment:
Do you need a "|" at the start of this line as well?

Reply:
Github only chokes when the first cell is empty - which is not the case here.